RecoTauTag/RecoTau/src/DeepTauBase.cc

```diff
@@ -81,4 +110,48 @@ void DeepTauBase::CreateOutputs(edm::Event& event, const tensorflow::Tensor& pre
     }
 }
 
+std::unique_ptr<DeepTauCache> DeepTauBase::initializeGlobalCache(const edm::ParameterSet& cfg )
+{
+    std::string graphName = edm::FileInPath(cfg.getParameter<std::string>("graph_file")).fullPath();
+    bool memMapped = cfg.getParameter<bool>("memMapped");
+    return std::make_unique<DeepTauCache>(graphName, memMapped );
+}
+
+// void DeepTauBase::globalEndJob(const DeepTauCache* cache) { }
```

Comment:
Please clean code with is commented out (here and in other places where it might apply).